packages/manager/modules/dedicated-universe-components/src/index.js

```diff
@@ -0,0 +1,96 @@
+import angular from 'angular';
+
+import ducBandwidth from './bandwidth';
+import ducBytes from './bytes';
+import ducContract from './contract';
+import ducNotification from './notification';
+import ducPrice from './price';
+import ducTabs from './tabs';
+import ducTranslate from './translate';
+
+import ducCheckboxSwitch from './directives/checkboxSwitch';
+import ducSshKeySwitch from './directives/sshkeySwitch';
+
+import config from './config';
+import user from './user';
+import noDefaultMeansOfPayment from './no-default-means-of-payment';
+import sshkeyMinFilter from './filters/sshkeyMin';
+import validatorService from './validator.service';
+
+const moduleName = 'ngOvhDedicatedUniverseComponents';
+
+angular
+  .module(moduleName, [
+    ducBandwidth,
+    ducBytes,
+    ducContract,
+    ducNotification,
+    ducPrice,
+    ducTabs,
+    ducTranslate,
+    ducCheckboxSwitch,
+    ducSshKeySwitch,
+    user,
+    noDefaultMeansOfPayment,
+  ])
+  .constant('constants', {
+    swsProxyRootPath: config.swsProxyRootPath,
+    aapiRootPath: config.aapiRootPath,
+    target: config.target,
+    renew: config.constants.RENEW_URL,
+    urls: config.constants.URLS,
+    UNIVERS: config.constants.UNIVERS,
+    TOP_GUIDES: config.constants.TOP_GUIDES,
+    vmsUrl: config.constants.vmsUrl,
+    travauxUrl: config.constants.travauxUrl,
+    aapiHeaderName: 'X-Ovh-Session',
+    vrackUrl: config.constants.vrackUrl,
+    REDIRECT_URLS: config.constants.REDIRECT_URLS,
+    DEFAULT_LANGUAGE: config.constants.DEFAULT_LANGUAGE,
+    FALLBACK_LANGUAGE: config.constants.FALLBACK_LANGUAGE,
+    SUPPORT: config.constants.SUPPORT,
+    billingRenew: config.constants.billingRenew,
+  })
+  .constant('website_url', config.constants.website_url)
+  .constant('BILLING_BASE_URL', 'billing/')
+  .constant('Billing.constants', {
+    aapiRootPath: config.aapiRootPath,
+    swsProxyRootPath: config.swsProxyRootPath,
+    paymentMeans: [
+      'bankAccount',
+      'paypal',
+      'creditCard',
+      'deferredPaymentAccount',
+    ],
+    target: config.target,
+  })
+  .constant('Billing.URLS', {
+    renew: config.constants.billingRenew,
+  })
+  .constant('UserAccount.constants', {
+    aapiRootPath: config.aapiRootPath,
+    swsProxyRootPath: config.swsProxyRootPath,
+    target: config.target,
+  })
+  .constant('CountryConstants', {
+    support: config.constants.URLS.support,
+  })
+  .constant('AccountCreationURLS', config.constants.accountCreation)
```

Comment:
If used in outside of the component we should prefix it and update usages in existing code:
```suggestion
  .constant('ducAccountCreationURLS', config.constants.accountCreation)
```

If use only in `dedicated-universe-components`, we could move it away from angularJS and use it as ES6 `const`.

_This comments is valid for all `constant/filter/service` in this file_

Reply:
I have removed this, it's not used.